spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/TaskConfigurationProperties.java

```diff
@@ -32,8 +40,12 @@
 	/**
 	 * The task application name to be used for the composed task runner.
 	 */
+	@NotBlank
 	private String composedTaskRunnerName = "composed-task-runner";
 
+	@Min(1)
+	private long maximumConcurrentTasks = Long.MAX_VALUE;
```

Comment:
I think this is the wrong default, while consistent with the behavior of the past servers, we are letting people accidentally get into a situation where they can exhaust all their AIs.  Picking a value such as 20 is probably a reasonable compromise.

Reply:
Updated in the subsequent PR